client/post-editor/editor-revisions-list/header.jsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+const EditorRevisionsListHeader = ( { loadRevision, selectedRevisionId, translate } ) => (
+	<div className="editor-revisions-list__header">
+		<Button
+			className="editor-revisions-list__load-revision"
+			compact={ true }
+			disabled={ selectedRevisionId === null }
+			onClick={ loadRevision }
+		>
+			{ translate( 'Load revision in the editor' ) }
+		</Button>
+	</div>
+);
+
+EditorRevisionsListHeader.propTypes = {
+	loadRevision: PropTypes.func,
+	selectedRevisionId: PropTypes.number,
+	translate: PropTypes.func,
```

Comment:
I've done a full pass on all prop types, if there is a usage where it can lead to a runtime exception (array/object access and function calls), I've `isRequired`-ify the prop type. Is there any "guideline" in calypso to choose between `isRequired` vs default props?